src/main/java/com/codingnomads/betty/logic/services/ProbabilityToWinService.java

```diff
@@ -0,0 +1,44 @@
+package com.codingnomads.betty.logic.services;
+
+import com.codingnomads.betty.logic.models.TeamProbabilityToWin;
+import com.codingnomads.betty.logic.models.TeamSentimentScore;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ProbabilityToWinService {
+
+    public TeamProbabilityToWin getProbabilityToWinFromSentimentAnalysis(TeamSentimentScore teamSentimentScore) {
+        if (teamSentimentScore.getScore() == null) {
+            throw new NullPointerException("Team Score is null");
+        }
+
+        return createTeamProbability(teamSentimentScore);
+    }
+
+    private TeamProbabilityToWin createTeamProbability(TeamSentimentScore teamSentimentScore) {
+
+        Integer teamScore = teamSentimentScore.getScore();
+
+        TeamProbabilityToWin teamProbabilityToWin = new TeamProbabilityToWin();
+        teamProbabilityToWin.setTeamName(teamSentimentScore.getTeamName());
+        teamProbabilityToWin.setMatchDateTime(teamSentimentScore.getMatchDateTime());
+        teamProbabilityToWin.setPlayingHome(teamSentimentScore.getPlayingHome());
+
+        return calculateTeamProbabilityToWin(teamScore, teamProbabilityToWin);
```

Comment:
Instead of passing the pojo `TeamProbabilityToWin` inside this mehtod, would be better to just do like: 

`teamProbabilityToWin.setProbabilityToWin(transformScoreToProbability(teamScore));`

Reply:
![tenor-94450224](https://user-images.githubusercontent.com/32600398/56576931-4678da00-65fc-11e9-8034-9284e34a3a62.gif)